ceph.spec.in

```diff
@@ -613,7 +613,7 @@ python-cephfs instead.
 # common
 #################################################################################
 %prep
-%autosetup -p1
+%setup -q -n %{name}-%{version}-%{release}
```

Comment:
Thanks for the update Kefu Chai. I will try the new changes to ensure it fixes the issue.